Library/Formula/ired.rb

```diff
@@ -0,0 +1,23 @@
+class Ired < Formula
+  desc "Minimalistic hexadecimal editor designed to be used in scripts."
+  homepage "http://www.radare.org/"
```

Comment:
Since that website doesn't have a dedicated page for this, maybe the GitHub page would be better as the `homepage`: https://github.com/radare/ired